virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc

```diff
@@ -8,18 +8,22 @@ toc::[]
 
 You can connect a virtual machine (VM) to an Open Virtual Network (OVN)-Kubernetes secondary network. The OVN-Kubernetes Container Network Interface (CNI) plug-in uses the Geneve (Generic Network Virtualization Encapsulation) protocol to create an overlay network between nodes.
 
-{VirtProductName} currently supports the flat layer 2 topology. This topology connects workloads by a cluster-wide logical switch. You can use this overlay network to connect VMs on different nodes, without having to configure any additional physical networking infrastructure.
+{VirtProductName} supports the flat layer 2 and localnet topologies. 
+
+* A layer 2 topology connects workloads by a cluster-wide logical switch. You can use this overlay network to connect VMs on different nodes, without having to configure any additional physical networking infrastructure.
+
+* A localnet topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) bridge on cluster nodes.
 
 To configure an OVN-Kubernetes secondary network and attach a VM to that network, perform the following steps:
 
-. Create a network attachment definition (NAD) by using the web console or the xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-creating-layer2-nad-cli_virt-connecting-vm-to-ovn-secondary-network[CLI].
+. xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#creating-ovn-nad[Configure an OVN-Kubernetes secondary network] by creating a network attachment definition (NAD).
 
-. Add information about the secondary network interface to the VM specification by using the web console or the xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-attaching-vm-to-ovn-secondary-nw-cli_virt-connecting-vm-to-ovn-secondary-network[CLI].
+. xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#attaching-vm-to-ovn-secondary-nw[Connect the VM to the OVN-Kubernetes secondary network] by adding the network details to the VM specification.
```

Comment:
IMO we are missing a procedure step to provision the kubernetes-nmstate NNCP; it should happen before provisioning the NAD.

Reply:
@maiqueb Does creating the NNCP step apply to both layer 2 and local net topologies? I have included it as a prerequisite in the [Creating a NAD for localnet topology using the CLI](https://70729--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network#virt-creating-localnet-nad-cli_virt-connecting-vm-to-ovn-secondary-network) procedure.

Reply:
> Does creating the NNCP step apply to both layer 2 and local net topologies? 

No, it's only for localnet. 

> I have included it as a prerequisite in the [Creating a NAD for localnet topology using the CLI](https://70729--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network#virt-creating-localnet-nad-cli_virt-connecting-vm-to-ovn-secondary-network) procedure.

Yes, I've noticed that. If that's how we'll roll (nothing against it), we should link the openshift networking documentation to help the user configure it. It's not exactly straight-forward IMO.